### DISCUSSION TITLE
Poll for new name

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/flow#94] [Polls]
See context [here](https://github.com/sablier-labs/v2-open-ended/discussions/83).

**Comment by andreivladbrg** [comment]
5 votes already, `Flow` has a majority and created an issue: https://github.com/sablier-labs/v2-open-ended/issues/95

**Comment by PaulRBerg** [comment]
I voted for Flow but a part of me cries because `Infinio` would have been a cool name.

**Comment by andreivladbrg** [comment]
everybody voted, closing

**Comment by smol-ninja** [comment]
In the repo, would it be called `SablierFlow` instead of `SablierV2OpenEnded`?

**Reply by PaulRBerg** [reply]
Correct
